NEWS.md

```diff
@@ -1,13 +1,26 @@
 # pkgnet 0.3.2.9999 (current dev)
 
 ## NEW FEATURES
-None
+
+* Objects of new `DirectedGraph` class now slot into the `pkg_graph` field of network reporters. These objects encapsulate the graph modeling of networks and have a more expressive set of methods for analysis. Check out the full documentation with `?DirectedGraph`.  ([#181](https://github.com/UptakeOpenSource/pkgnet/pull/181))
+    * Use `pkg_graph$node_measures` and `pkg_graph$graph_measures` to respectively calculate node-level and graph-level measures. 
+    * Use `pkg_graph$default_node_measures` and `pkg_graph$default_graph_measures` to see the measures calculated by default. 
+    * Use `pkg_graph$available_node_measures` and `pkg_graph$available_graph_measures` to see the all supported measures.
+    * The igraph object is now instead available at `pkg_graph$igraph`.
 
 ## CHANGES
-None
+
+* Standardizing on the language "dependency" and "reverse dependency" to describe the directed graph relationships in the package. This completes the change introduced in v0.3.0 where edge direction convention was set to point in the direction of dependency. So this means that "depend on" follows the edge arrow direction, and "reverse depends on" is reverse edge arrow direction. ([#191](https://github.com/UptakeOpenSource/pkgnet/issues/106), [#181](https://github.com/UptakeOpenSource/pkgnet/pull/181))
```

Comment:
@jayqi I'm taking a quick run though this now, but will you please `@` me when it's rebased to `master` so I can focus on just the new code introduced in this PR?